src/hooks/trips/utils.ts

```diff
@@ -0,0 +1,129 @@
+import type { BasePayload } from 'payload';
+import { Types } from 'mongoose';
+import { Trip } from '@/payload-types';
+
+export const generateTripCustomers = async (trip: Trip, payload: BasePayload) => {
+
+  const areaIds = trip.areas.map(a =>
```

Comment:
move this file from hook, can create new folder besides `hooks` and call it `aggregations/trips.ts`.
Also implement `coderabbitai` feedback